### ISSUE TITLE
Security header STSIncludeSubDomains is not set correctly

### ISSUE THREAD

**Issue by CVJoint** [containous/traefik#5323]
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [x] No

### What did you do?

Running in docker:
`- traefik.http.middlewares.secure-headers.headers.STSIncludeSubdomains=true`

### What did you expect to see?

After a scan at https://securityheaders.com I expect to see `includeSubDomains` with a capital "D", and for this item to appear in bold.

**preload**, for example, is in bold.

### What did you see instead?

This header is not in bold and is listed as `includeSubdomains`

Reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security

### Output of `traefik version`:

I'm using docker image: `traefik:v2.0.0-rc2`

```
$ docker run traefik version
Version:      v1.7.14
Codename:     maroilles
Go version:   go1.12.8
Built:        2019-08-14_09:46:58AM
OS/Arch:      linux/amd64
```

**Comment by SantoDE** [comment]
Hey @CVJoint,

thanks for your response. Just to be sure: You are really running 2..0.0-rc2, correct? Because your CLI output mentions **v1.7.14*.

If you're really running the v2, please provide us with a copy of your 

> /api/rawdata

 if possible.

**Comment by dtomcej** [comment]
May be related to #5397

**Comment by dtomcej** [comment]
Hello @CVJoint ,

The RFC for the security header clearly states: (https://tools.ietf.org/html/rfc6797#page-15)

> 3.  Directive names are case-insensitive.

However, despite this, I have opened a PR to update the casing of that directive, and once merged, we will update it here.

https://github.com/unrolled/secure/pull/59

**Comment by traefiker** [comment]
Closed by #5492.
